src/otx/algorithms/detection/configs/instance_segmentation/convnext_maskrcnn/template.yaml

```diff
@@ -0,0 +1,62 @@
+# Description.
```

Comment:
This should be renamed to template_experimental.yaml since we haven't yet agreed on adding this model to Geti.